gradio/components.py

```diff
@@ -3930,6 +3930,172 @@ def style(
         return self
 
 
+@document("style")
+class AnnotatedImage(Selectable, IOComponent, JSONSerializable):
+    """
+    Displays a base image and colored subsections on top of that image. Subsections can take the from of rectangles (e.g. object detection) or masks (e.g. image segmentation).
+    Preprocessing: this component does *not* accept input.
+    Postprocessing: expects a {Tuple[numpy.ndarray | PIL.Image | str, List[Tuple[numpy.ndarray | Tuple[int, int, int, int], str]]]} consisting of a base image and a list of subsections, that are either (x1, y1, x2, y2) tuples identifying object boundaries, or 0-1 confidence masks of the same shape as the image. A label is provided for each subsection.
+
+    Demos: image_segmentation
+    """
+
+    def __init__(
+        self,
+        value: Tuple[
+            np.ndarray | _Image.Image | str,
+            List[Tuple[np.ndarray | Tuple[int, int, int, int], str]],
+        ]
+        | None = None,
+        *,
+        show_legend: bool = True,
```

Comment:
I don't think this parameter has any effect. Setting it to `False` still displayed the legend

Reply:
fixed